src/pages/Home.vue

```diff
@@ -96,6 +96,7 @@ export default {
     return {
       product_list: [],
       pageLoading: false,
+      moonlight_URL: window.env?.fp_api_server || 'https://api.fynd.com'
```

Comment:
Add static URL of production only @jinalviranii

Reply:
Use generic name like documentation_url @jinalviranii

Reply:
Used production documentation static link